.github/workflows/docker-release.yml

```diff
@@ -34,14 +34,14 @@ jobs:
           release_tag=${{ steps.get-latest-tag.outputs.tag }}
           echo "LATEST_VERSION=${release_tag#*/}" >> "$GITHUB_ENV"
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@aa33708b10e362ff993539393ff100fa93ed6a27
+        uses: docker/setup-buildx-action@4fd812986e6c8c2a69e18311145f9371337f27d4
```

Comment:
If I close this PR and re-run the github action. Will the automation start from master or will it re-open this branch?
I want to avoid having to go through all files manually to verify if they're merged correctly

Reply:
this is a custom GH action and not a dependabot one, if you close I belive you can run it manually from https://github.com/mongodb/mongodb-atlas-cli/actions/workflows/autoupdate-sdk.yaml